assets/privacy.md

```diff
@@ -0,0 +1,53 @@
+# Privacy Policy
+
+### About us
+
+[My Free-time in a week](https://myfreetimeinaweek.in) (the "Application") is stand-alone webpage created by an individual developer named Avikalp Kumar Gupta, a resident of Bengaluru, India. This Application was created with the objective of helping people reassess how they spend their time in order to either be more productive, find time for their hobbies or their families or even just be happier with their lifestyle.
+
+This Application has been created as a utility tool, and runs entirely on the client-side. Hence, our servers do not collect any information. However, we do use [Google Adsense](https://www.google.com/adsense/start/) to show advertisement on our website. Hence, our privacy policy is identical to Google's policies on serving ads, and hence is not in our sole control. This Privacy Policy attempts to describe how your personal information is collected, used, and shared when you visit [My Free-time in a week](https://myfreetimeinaweek.in) (the “Application”).
+
+### Personal information that we collect
+
+When you visit the Application, we automatically collect certain information about your device, including information about your web browser, IP address, time zone, and some of the cookies that are installed on your device. Additionally, as you use the Application, we collect information about the individual web pages or products that you view, what websites or search terms referred you to the Application, and information about how you interact with the Application. We refer to this automatically-collected information as “Device Information.”
+
+We collect Device Information using the following technologies:
+
+- “Cookies” are data files that are placed on your device or computer and often include an anonymous unique identifier. For more information about cookies, and how to disable cookies, visit http://www.allaboutcookies.org.
+- “Log files” track actions occurring on the Application, and collect data including your IP address, browser type, Internet service provider, referring/exit pages, and date/time stamps.
+- “Web beacons,” “tags,” and “pixels” are electronic files used to record information about how you use the Application.
+
+When we talk about “Personal Information” in this Privacy Policy, we are talking about the Device Information.
+
+### HOW DO WE USE YOUR PERSONAL INFORMATION?
+
+We use the Device Information that we collect to help us screen for potential risk and fraud (in particular, your IP address), and more generally to improve and optimize our Application (for example, by generating analytics about how our customers browse and interact with the Application, and to assess the success of our marketing and advertising campaigns). This information is used by Google Adsense to personalize and target advertisements displayed on this Application.
```

Comment:
A lot of this information is actually incorrect. We do not collect any information in reality. All the information is only collected by Google Adsense.